app/frontends/ogc-coverages/util/parameter-parsing.ts

```diff
@@ -160,6 +160,44 @@ function _getDimensionName(value: string): string {
   }
 }
 
+/**
+ * Parses the provided point parameters and ensures they are valid, throwing an error message
+ * if not
+ *
+ * @param values - An array of all the specified point parameters from the request
+ * @param dimConfig - A mapping of dimension names to min, max, and data type values,
+ *   see `dimensionInfo` (the default value) in this file.  Usually should not be specified,
+ *   except for testing.
+ * @returns An array with two elements corresponding to [longitude, latitude]
+ * @throws ParameterParseError - if a subset parameter cannot be parsed, has unrecognized
+ *   axis names, or is otherwise invalid
+ */
+export function parsePointParam(
+  values: string,
```

Comment:
This type is incorrect - values is an Array of numbers as defined in the OGC schema for point.